SPECS/python-retrying/python-retrying.spec

```diff
@@ -6,14 +6,18 @@ written in Python, to simplify the task of adding retry behavior to\
 just about anything.\
 
 Name:           python-%{pypi_name}
-Version:        1.3.3
-Release:        6%{?dist}
+Version:        1.3.4
+Release:        1%{?dist}
 Summary:        General-purpose retrying library in Python
 License:        ASL 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
-URL:            https://github.com/rholder/retrying
-Source0:        %{url}/archive/refs/tags/v%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
+URL:            https://pypi.org/project/retrying
+Source0:        https://files.pythonhosted.org/packages/ce/70/15ce8551d65b324e18c5aa6ef6998880f21ead51ebe5ed743c0950d7d9dd/retrying-1.3.4.tar.gz
+
+# The tarball does not contain the tests, so add them here
+Source1:        test_retrying.py
```

Comment:
Just curious, Is this the way new tests are expected to be added? Does it need to somehow be packaged into the source or anything like that?

Reply:
probably easier and more clear than creating a patch file that injects the test into the source code

Reply:
The test file is the part of the source but not present in the tarball that I download. So here I am adding that separately to the source code. This will run the tests with pytest.